internal/cephfs/cephfs_util.go

```diff
@@ -23,6 +23,8 @@ import (
 	"github.com/ceph/ceph-csi/internal/util"
 )
 
+const MetadataPoolType = "metadata"
```

Comment:
where this is getting used? do we need this to be exported?

Reply:
Ah, it was used in a previous version where go-ceph did not have `ListFileSystems()` that contains the `MetadataPool` attribute. It is not used anymore. I'll remove it.